doc/CaseStudies.md

```diff
@@ -84,3 +84,19 @@ Looking at the [Fuzz blockers](https://storage.googleapis.com/oss-fuzz-introspec
 the culprit was that `file` needs to be passed data via a proper file descriptor
 to exercise its elf-related codepath, and thus a file-based fuzzer was [promptly added](https://github.com/google/oss-fuzz/pull/8542),
 [bumping the coverage close to 90%]( https://storage.googleapis.com/oss-fuzz-introspector/file/inspector-report/20220930/fuzz_report.html ).
+
+# [bzip2](https://storage.googleapis.com/oss-fuzz-introspector/bzip2/inspector-report/20221016/fuzz_report.html)
+
+Introspector report: [link](https://storage.googleapis.com/oss-fuzz-introspector/bzip2/inspector-report/20221016/fuzz_report.html)
+
+Bzip2's current fuzzers call the compression and decompression functions, but the file handling code is not being addressed.  This is seen
+by the suggestions in the remaining optimal interesting functions.
+
+![interest](https://user-images.githubusercontent.com/5808510/196543962-b65e4197-6f74-4d2c-b6f9-e12d3a9be034.png)
+
+Based on these hints, a [new file-based fuzzer](https://github.com/google/oss-fuzz/pull/8790) was added to hit [these recommended functions.](https://github.com/libarchive/bzip2/blob/master/bzlib.c)  Only a single fuzzer was needed to create a temporary file, open it via a file descriptor, then write to and then read from this file.
+
+![calltree](https://user-images.githubusercontent.com/5808510/196543332-4c0554ac-43e9-4c16-8452-df79524608a8.png)
+
+The addition of this fuzzer increases the project's coverage to 80%.
```

Comment:
We can wait another day to have the new reports generated and link it here.

Reply:
Here we go: https://storage.googleapis.com/oss-fuzz-introspector/bzip2/inspector-report/20221019/fuzz_report.html

Please use this report to provide more information on the improvements.